python/sdk/client/api/__init__.py

```diff
@@ -1,17 +1,3 @@
-# Copyright 2020 The Merlin Authors
```

Comment:
Is it intended that all copyright header are removed?

Reply:
For Python generated API code, yes. Previously the copyright header is not generated by Swagger Codegen